src/maidsafe/vault/mpid_manager/value.h

```diff
@@ -0,0 +1,57 @@
+/*  Copyright 2015 MaidSafe.net limited
+
+    This MaidSafe Software is licensed to you under (1) the MaidSafe.net Commercial License,
+    version 1.0 or later, or (2) The General Public License (GPL), version 3, depending on which
+    licence you accepted on initial access to the Software (the "Licences").
+
+    By contributing code to the MaidSafe Software, or to this project generally, you agree to be
+    bound by the terms of the MaidSafe Contributor Agreement, version 1.0, found in the root
+    directory of this project at LICENSE, COPYING and CONTRIBUTOR respectively and also
+    available at: http://www.maidsafe.net/licenses
+
+    Unless required by applicable law or agreed to in writing, the MaidSafe Software distributed
+    under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+    OF ANY KIND, either express or implied.
+
+    See the Licences for the specific language governing permissions and limitations relating to
+    use of the MaidSafe Software.                                                                 */
+
+#ifndef MAIDSAFE_VAULT_MPID_MANAGER_VALUE_H_
+#define MAIDSAFE_VAULT_MPID_MANAGER_VALUE_H_
+
+#include <cstdint>
+#include <string>
+#include <vector>
+
+#include "maidsafe/common/data_types/data_name_variant.h"
+
+#include "maidsafe/vault/types.h"
+
+namespace maidsafe {
+
+namespace vault {
+
+class MpidManagerValue {
+ public:
+//  MpidManagerValue();
```

Comment:
I think the current favoured approach is to explicitly specify all compiler-generated class functions as `= delete`, `=default` or user-defined if neither of these apply (this includes the default constructor and the destructor).  The exception is that MSVC currently can't handle move constructors or move assignment operators being defined as `= default` - we need to write these by hand for now.